### PR TITLE
v1.21.0

### PR DESCRIPTION
**Pull request by aminlatifi**
- Solana SPL token support
- Import backup service saved donations
- Bug fixes

### CODE REVIEW

File: src/repositories/donationRepository.ts [hunk 1]
```diff
@@ -387,12 +387,20 @@ export const getPendingDonationsIds = (): Promise<{ id: number }[]> => {
...
+    where: [
+      {
+        status: DONATION_STATUS.PENDING,
+        isFiat: false,
+        createdAt: MoreThan(date),
+      },
+      {
+        status: DONATION_STATUS.PENDING,
+        isFiat: false,
+        importDate: MoreThan(date),
```
Comment (CarlosQ96) [inline src/repositories/donationRepository.ts:401]: Is this WHERE correct? Its the same expression twice. Maybe one of them is LessThan?

**Review by CarlosQ96 — APPROVED** [review]
Thanks! LGTM!

File: src/repositories/donationRepository.ts (hunk 1 above)
Comment (CarlosQ96) [reply src/repositories/donationRepository.ts:401]: Nvm its ok Thanks for clarifying in chat

**Review by mohammadranjbarz — APPROVED** [review]
Thanks @aminlatifi  and @CarlosQ96  LGTM

**Pull request merged**